bootstrap-osx.sh

```diff
@@ -28,6 +28,8 @@ cp /usr/local/lib/libav* libs/lib
 cp /usr/local/lib/libswscale* libs/lib
 cp /usr/local/lib/libqrencode* libs/lib
 cp /usr/local/lib/libsqlcipher* libs/lib
+cp /usr/local/lib/libfilteraudio* libs/lib
+cp /usr/local/lib/libtox* libs/lib
```

Comment:
No.

filter_audio isn't even installed in the system, so trying to copy it from there is simply wrong.

On older version of scripts might have used to install it system-wide, I don't really remember.